addons/pos_restaurant/static/src/overrides/models/pos_store.js

```diff
@@ -283,7 +283,11 @@ patch(PosStore.prototype, {
             if (orders.length > 0) {
                 this.set_order(orders[0]);
                 this.orderToTransferUuid = null;
-                this.showScreen(orders[0].get_screen_data().name);
+                const props = {};
+                if (orders[0].get_screen_data().name === "PaymentScreen") {
+                    props.orderUuid = orders[0].uuid;
+                }
+                this.showScreen(orders[0].get_screen_data().name, props);
```

Comment:
this code looks familiar. Can you check if it can be extracted?

Reply:
Hello @caburj 

[REF](https://github.com/odoo/odoo/pull/180038/files#diff-a29b5d2c83a83cefb22ee666b5b7a164e5548d070800b8f93ef1f7beafff3e3eR103)

For the currentOrder getter in the paymentScreen, the orderUuid is required as props.
In the referenced PR, while opening the floating order orderUuid was passed, but when opening the order from the floorscreen, it was left out.

Thanks!